### PR TITLE
fix: Add autodevops action.

### PR DESCRIPTION
**Pull request by gary-van-woerkens**

### CODE REVIEW

**Review by douglasduteil — APPROVED** [review]
🌶️

**Review by revolunet — APPROVED** [review]
manque plus que la doc :)

File: autodevops/action.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  using: "composite"
+  steps:
+
+  - name: Use autodevops build and register
+    uses: SocialGouv/actions/autodevops-build-register@master
+    with:
+      token: ${{ inputs.token }}
+      project: ${{ inputs.project }}
+      imageName: ${{ inputs.imageName }}
+      dockerfile: ${{ inputs.dockerfile || './Dockerfile' }}
```
Comment (douglasduteil) [inline autodevops/action.yml:29]: as default ?

File: autodevops/action.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    description: "The deployment environment (dev | preprod | prod)"
+  productionNamespace:
+    description: "Override production namesapce"
+
+runs:
+  using: "composite"
+  steps:
+
+  - name: Use autodevops build and register
+    uses: SocialGouv/actions/autodevops-build-register@master
```
Comment (douglasduteil) [inline autodevops/action.yml:24]: can be relatif to follow the version of the running action ?

Comment (douglasduteil) [reply autodevops/action.yml:24]:
```suggestion
    uses: ./autodevops-build-register@master
```

File: autodevops/action.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    with:
+      token: ${{ inputs.token }}
+      project: ${{ inputs.project }}
+      imageName: ${{ inputs.imageName }}
+      dockerfile: ${{ inputs.dockerfile || './Dockerfile' }}
+
+  - name: Use autodevops manifests generation
+    uses: SocialGouv/actions/autodevops-manifests@master
```
Comment (douglasduteil) [inline autodevops/action.yml:32]:
```suggestion
    uses: ./autodevops-manifests@master
```

File: autodevops/action.yml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    with:
+      environment: ${{ inputs.environment }}
+      productionNamespace: ${{ inputs.productionNamespace }}
+
+  - name: Use autodevops deployment
+    uses: SocialGouv/actions/autodevops-deploy@master
```
Comment (douglasduteil) [inline autodevops/action.yml:38]:
```suggestion
    uses: ./autodevops-deploy@master
```

File: autodevops/action.yml (hunk 2 above)
Comment (gary-van-woerkens) [reply autodevops/action.yml:24]: Nope, I think I tested it and it didn't work :(

File: autodevops/action.yml (hunk 1 above)
Comment (gary-van-woerkens) [reply autodevops/action.yml]: Yes, done.

**Pull request merged**